include/nbl/asset/IMeshPackerV2.h

```diff
@@ -29,6 +29,10 @@ class IMeshPackerV2Base
         uint16_t floatArrayElementsCnt = 0u;
         uint16_t intArrayElementsCnt = 0u;
         uint16_t uintArrayElementsCnt = 0u;
+        bool isUintBufferUsed = false;
+        bool isUvec2BufferUsed = false;
+        bool isUvec3BufferUsed = false;
+        bool isUvec4BufferUsed = false;
```

Comment:
btw did we establish whether SSBO and UTB usage of a V2 packer are mutually exclusive ?